internal/tasks/log.go

```diff
@@ -31,12 +31,12 @@ var sendEventsToSentry bool
 
 func init() {
 	dsn := os.Getenv("CASTELLUM_SENTRY_DSN")
-	sendEventsToSentry = (dsn != "")
-	if sendEventsToSentry {
+	if dsn != "" {
 		err := sentry.Init(sentry.ClientOptions{Dsn: dsn})
 		if err != nil {
-			logg.Error("Sentry initialization failed: %s", err.Error())
+			logg.Fatal("Sentry initialization failed: %s", err.Error())
```

Comment:
Please revert this line change. Castellum should not be refusing service just because Sentry is off.